src/components/dialogs/addUser/AddUserDialog.tsx

```diff
@@ -40,6 +43,21 @@ export const AddUserDialog:React.FC<Props> = ({ open, closeDialog }) => {
       user_name: data.name,
       email: data.email
     });
+    closeDialog();
+  }
+
+  const clear = (
+    setValues: (values: FormData) => void,
+    setTouched: (touched: FormikTouched<FormData>) => void,
+    setErrors: (errors: FormikErrors<FormData>) => void,
+  ) => {
+    setValues(initialValues);
+    setTouched({})
```

Comment:
I think you don't need this function

Reply:
Why?

Reply:
If I remove `setErrors` instead of `setTouched`, and everything works fine.

Reply:
can you explain why do you use `setErrors` function and `setTouched`?